online/src/main/scala/ai/chronon/online/FetcherBase.scala

```diff
@@ -264,11 +268,12 @@ class FetcherBase(kvStore: KVStore,
     * @return the GroupByServingInfoParsed containing the latest serving information.
```

Comment:
Added one comment here for the return param 
```suggestion
    * @return the GroupByServingInfoParsed containing the latest serving information.
    * @return the maximum ts from the batch IR responses. It will be later used for filtering the streaming rows after the such ts to avoid any potential duplicate records between the batch IR and streaming rows. 
```